cluster-install/install.sh

```diff
@@ -1328,7 +1359,7 @@ create_kubectl_patch_for_containers() {
 exit_if_problems() {
     if [ ${#_PROBLEMS[@]} -ne 0 ]; then
         debug_is_on && print_built_config
-        logheader $__ERROR "Pre-flight check failed! Please resolve the following issues and try again:"
+        logheader $__ERROR "Cluster management failed! We've identified the following issues:"
```

Comment:
Reworded this since it's no longer just a **Pre-flight** check script, and the wording on getting the user to resolve the issues kind of sounded like we're blaming them 😅